ci-scripts/build-for-deploy.sh

```diff
@@ -4,7 +4,7 @@ set -o pipefail
 
 echo "-----"
 echo "Building SPA libs"
-yarn build:core:lib
+yarn build:core:lib && yarn build:core:lib:cds
```

Comment:
I believe it would only be yarn build:core:lib:cds

Reply:
🤦‍♂ of course, `build:core:lib:cds ` already builds `core:lib` first